.github/workflows/nodejs.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           fetch-depth: 0
 
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
```

Comment:
we need to update this on [line 80](https://github.com/webpack-contrib/html-loader/pull/501/files#diff-e98936aa52a6dd7416e4296e9628456227d834f7245967383fd9ff80fd985dadR80) as well